Database_Structures.md

```diff
@@ -45,3 +45,23 @@ fk_questionid|
 option_content|
 picture_link|
 qustion_multiplier|
+
+USER|
+-|
+id|
+email|
+username|
+registered_at|
+
+TOKEN|
+-|
+value|
+created_at|
+expires_at|
+fk_userId|
+
+EXPIRED_TOKEN|
```

Comment:
What's the purpose of storing expired tokens?

Reply:
I was thinking it would be a good idea to keep token for some time due to retention reasons. For example, if user were to reset their password it would be wise to expire all his existing tokens for security reasons. However, as @lukasrakauskas pointed out, tokens are not really required to be persisted, so I will delete this shema.